packages/components/src/angle-picker-control/index.tsx

```diff
@@ -57,6 +57,7 @@ function UnforwardedAnglePickerControl(
 		<Flex { ...restProps } ref={ ref } className={ classes } gap={ 2 }>
 			<FlexBlock>
 				<NumberControl
+					__next40pxDefaultSize
```

Comment:
FYI, either having the `size="__unstable-large"` or `__next40pxDefaultSize` is good enough, so we can remove one of them. (`size="__unstable-large"` was an older way to opt into this size 😄)

Reply:
This is now addressed, removed `size="__unstable-large"` and used the new way to add the size.